src/units/abilities.cpp

```diff
@@ -215,6 +215,17 @@ bool unit::get_ability_bool(const std::string& tag_name, const map_location& loc
 		}
 	}
 
+	for(const unit& unit_itor : units){
+		if (!unit_itor.affect_distant(tag_name) || unit_itor.incapacitated() || &(unit_itor) == this) {
```

Comment:
`&(unit_itor)` should be just `&unit_itor`. Please don't add extra parenthesis around things.

Reply:
done